<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Menu" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>691</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>-10</y>
       <width>331</width>
       <height>121</height>
      </rect>
     </property>
     <widget class="QWidget" name="NouveauDeckPage">
      <widget class="QLineEdit" name="NomDeck">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom du deck</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ValiderNomDeck">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QLabel" name="VerifDeckName">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="QuitterAjouterDeck">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4"/>
    </widget>
    <widget class="QPushButton" name="FermerMain">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QWidget" name="AjouterCartePage">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>160</width>
         <height>133</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="NomCarteAAjouter">
            <property name="text">
             <string>Nom de la carte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NomDeckPourcarte">
            <property name="text">
             <string>Indiquer son deck</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CarteQuestionAAjouter">
            <property name="text">
             <string>Question </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CarteReponseAAjouter">
            <property name="text">
             <string>Réponse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="ValiderAjoutCarte">
            <property name="text">
             <string>Valider</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FermerAjoutCartePage">
            <property name="text">
             <string>Fermer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_14"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>97</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="NouveauDeckBouton">
        <property name="text">
         <string>Nouveau deck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SupprimerDeckBouton">
        <property name="text">
         <string>Supprimer deck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OuvrirDeckBouton">
        <property name="text">
         <string>Ouvrir Deck/Carte</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>341</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="VoirReponseBouton">
          <property name="text">
           <string>Voir réponse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CacherReponseBouton">
          <property name="text">
           <string>Cacher réponse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CarteParcourirBouton">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="AjouterCarteBouton">
          <property name="text">
           <string>Ajouter </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ModifierCarteBouton">
          <property name="text">
           <string>Modifier </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeplacerCarteBouton">
          <property name="text">
           <string>Déplacer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SupprimerCarteBouton">
          <property name="text">
           <string>Supprimer </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>51</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="LabelDeck">
        <property name="text">
         <string>Deck:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelNom">
        <property name="text">
         <string>Nom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelQuestion">
        <property name="text">
         <string>Question:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelReponse">
        <property name="text">
         <string>Réponse:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="CarteDeck">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>113</y>
       <width>319</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="CarteReponse">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>219</y>
       <width>319</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="CarteQuestion">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>186</y>
       <width>319</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CarteNom">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>319</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
